cmd/translate/service/service.go

```diff
@@ -78,18 +78,37 @@ var rootCmd = &cobra.Command{
 			log.Panicf("create new repo: %v", err)
 		}
 
-		googleTranslate, closeTranslate, err := fuzzy.NewGoogleTranslate(ctx, fuzzy.WithDefaultClient(ctx))
-		if err != nil {
-			log.Fatalf("create new google translate client: %v\n", err)
+		var (
+			translator    fuzzy.Translator
+			errTranslator error
+		)
+
+		translatorString := viper.GetString("service.translator")
+
+		switch translatorString {
```

Comment:
A better way is to avoid adding type in the variable name if you don't know the right answer. Alternatives are `translatorName`, `translatorType`, `translatorService`, `translatorAPI` etc. Personally, I would not declare a variable.

Also, we'll try to decouple viper.

 ```suggestion
		switch viper.GetString("service.translator") {
```